cmd/pow/cmd/ffs_get.go

```diff
@@ -3,18 +3,23 @@ package cmd
 import (
 	"context"
 	"errors"
+	"fmt"
 	"io"
 	"os"
 	"path"
 
 	"github.com/caarlos0/spin"
 	"github.com/ipfs/go-cid"
+	files "github.com/ipfs/go-ipfs-files"
+	ipfspath "github.com/ipfs/interface-go-ipfs-core/path"
 	"github.com/spf13/cobra"
 	"github.com/spf13/viper"
 )
 
 func init() {
 	ffsGetCmd.Flags().StringP("token", "t", "", "token of the request")
+	ffsGetCmd.Flags().String("ipfsrevproxy", "/ip4/127.0.0.1/tcp/6003", "Powergate IPFS reverse proxy multiaddr")
```

Comment:
Think we should make this flag (and the one in `stage`) normal `host:port` just to be consistent with how we specify the server address.

Reply:
Sure, agree. Will change.